androidsdk/src/main/java/com/wootric/androidsdk/objects/Settings.java

```diff
@@ -416,6 +434,12 @@ public static Settings fromJson(JSONObject settingsObject) throws JSONException
         Settings settings = new Settings();
         settings.firstSurvey = settingsObject.optLong("first_survey");
         settings.timeDelay = settingsObject.optInt("time_delay");
+        settings.userID = settingsObject.optLong("user_id");
```

Comment:
Do the same check that you did for `account_id` below since `user_id` will eventually be phased out.